startup.py

```diff
@@ -0,0 +1,26 @@
+import dbmanager
+import os
+
+print("\t*****************************")
+print("\t***** Choose an option ******")
+print("\t*****************************")
+print("\n")
+print("[1] Start up the database")
+print("[2] Stop the database instance")
+print("[3] Initialize the SDA database and tables")
+print("[q] Quite")
+
+choice = ''
+while choice != 'q':
+    choice = input("option: ")
+
+    if choice == 'q':
+        print("GoodBye!")
+    elif choice == '1':
+        dbmanager.startUp()
+    elif choice == '2':
+        dbmanager.stop()
+    elif choice == '3':
+        os.system("python3 initdb.py")
```

Comment:
No the script is fine, my issue was with Python using a system command to call Python. That doesn't make sense in this situation. Once @dlowrey gets the module set up right we can import it and use it normally.